cmd/observe/flows.go

```diff
@@ -570,6 +573,15 @@ func handleFlowArgs(writer io.Writer, ofilter *flowFilter, debug bool) (err erro
 	default:
 		return fmt.Errorf("invalid output format: %s", formattingOpts.output)
 	}
+	if !jsonOut {
+		if len(experimentalOpts.fieldMask) > 0 {
+			return fmt.Errorf("%s output format is not compatible with custom field mask", formattingOpts.output)
+		}
+		if len(experimentalOpts.fieldMask) == 0 && experimentalOpts.useDefaultMasks {
```

Comment:
nit: because of the previous check, the first condition can never be true
```suggestion
		if experimentalOpts.useDefaultMasks {
```